### PR TITLE
Enable Search in Playlist View

### PR DESCRIPTION
**Pull request by turtlepod**
### Description of the Change

Currently the plugin do not have playlist search feature.
This PR enable search in playlist view.

### Benefits

Currently this plugin only load 20 playlist item with no pagination.
This will allow user to search playlist and make this plugin more useful.

### Verification Process

fairly simple code improvement.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@oscarssanchez did you want to review / merge this into the 1.7.2 release or wait for the next release?

**Comment by oscarssanchez** [comment]
Hi @jeffpaul ,

I will test, review and schedule for the next release :)

File: includes/api/class-bc-cms-api.php [hunk 1]
```diff
@@ -172,11 +172,16 @@ public function playlist_get( $playlist_id ) {
 	 *
 	 * @since 1.0.0
 	 *
+	 * @param int $query Keyword Search Query.
```
[the remark this answers is not in the document]
Comment (turtlepod) [reply includes/api/class-bc-cms-api.php]: yup. my bad. will work on this today

**Comment by turtlepod** [comment]
@oscarssanchez Sorry for the late update.
Updated. Let me know if adjustment needed.

File: includes/api/class-bc-cms-api.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (oscarssanchez) [reply includes/api/class-bc-cms-api.php]: Thanks @turtlepod , appreciate it, this looks good!

**Comment by oscarssanchez** [comment]
Thanks for the changes @turtlepod , they look and behave good. It looks like there is some merge conflicts with the minified js. Can you pull the latest develop into your branch and run grunt? this should solve those issues.

**Pull request merged**